src/pages/projects/actions/DeleteProjectBtn.tsx

```diff
@@ -41,26 +43,24 @@ const DeleteProjectBtn: FC<Props> = ({ project }) => {
 
   return (
     <ConfirmationButton
+      onHoverText="Delete project"
+      toggleAppearance={isDeleteIcon ? "base" : "default"}
+      className="u-no-margin--bottom"
       isLoading={isLoading}
-      icon="delete"
       title="Confirm delete"
-      toggleCaption="Delete"
+      toggleCaption={isDeleteIcon ? undefined : "Delete project"}
       confirmMessage={
         <>
           Are you sure you want to delete the project{" "}
           <ItemName item={project} bold />?{"\n"}This action cannot be undone,
           and can result in data loss.
         </>
       }
-      confirmButtonLabel={
-        project.name === "default"
-          ? "The default project can't be deleted"
-          : "Delete"
-      }
+      confirmButtonLabel="Delete"
```

Comment:
I saw yesterday in another app they used "Delete permanently" which I think is better and can replace our extra sentence "This action cannot be undone, and can result in data loss". Wdyt, shall we adopt it here and in other delete buttons?

Reply:
I don't have a strong opinion for either, but I think the extra sentence is more noticeable than a "Delete permanently" button or title. Anyway, since it involves updating all the delete buttons, it's probably better suited for a separate effort.